docs/source/api/pipelines/stable_diffusion_2.mdx

```diff
@@ -24,7 +24,7 @@ For more details about how Stable Diffusion 2 works and how it differs from Stab
 
 ### Available checkpoints:
 
-Note that the architecture is more or less identical to [Stable Diffusion 1](./api/pipelines/stable_diffusion) so please refer to [this page](./api/pipelines/stable_diffusion) for API documentation.
+Note that the architecture is more or less identical to [Stable Diffusion 1](./api/pipelines/stable_diffusion/overview) so please refer to [this page](./api/pipelines/stable_diffusion/overview) for API documentation.
```

Comment:
Could we maybe add new sections for every example here?:

#### Text-to-Image

*Text-to-Image (512x512 resolution)*: [stabilityai/stable-diffusion-2-base](https://huggingface.co/stabilityai/stable-diffusion-2-base) with [`StableDiffusionPipeline`]
*Text-to-Image (768x768 resolution)*: [stabilityai/stable-diffusion-2](https://huggingface.co/stabilityai/stable-diffusion-2) with [`StableDiffusionPipeline`]

```python
```python
from diffusers import DiffusionPipeline, DPMSolverMultistepScheduler
import torch
repo_id = "stabilityai/stable-diffusion-2-base"
pipe = DiffusionPipeline.from_pretrained(repo_id, torch_dtype=torch.float16, revision="fp16")
pipe.scheduler = DPMSolverMultistepScheduler.from_config(pipe.scheduler.config)
pipe = pipe.to("cuda")
prompt = "High quality photo of an astronaut riding a horse in space"
image = pipe(prompt, num_inference_steps=25).images[0]
image.save("astronaut.png")
```

#### Image Inpainting

etc...

This way we could better link to the different tasks.

Reply:
for the upscaler should we use "Super-Resolution" or "Image-Upscaling"? I want to make the names consistent at least on the navigation bar

Reply:
"Super-Resolution" sounds good to me! Def in favor of making names consistent